### PR TITLE
8.0 update docs

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: openerp/openupgrade/doc/source/intro.rst [hunk 1]
```diff
@@ -13,8 +14,8 @@ https://github.com/OpenUpgrade/OpenUpgrade
 These branches are copies (forks in Git terminology) of the Odoo main project,
 but with extra commits that include migration scripts for each module.  One
 migration script updates that part of a database that is governed by the module
-for which it is written.  We use the upgrade native mechanism in Odoo, which
-was used in older versions of the Odoo/OpenERP server and presumably is still
+for which it is written.  We use the native upgrade mechanism in Odoo, which
+was used in older versions of the Odoo/Odoo server and presumably is still
```
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/intro.rst]: You can keep OpenERP here.

**Comment by rvalyi** [comment]
:+1: LGTM

File: openerp/openupgrade/doc/source/status.rst [hunk 2]
```diff
@@ -4,13 +4,13 @@ Status of the components
...
-  * An OpenERP module *OpenUpgrade Records* to generate and compare
+  * An Odoo module *OpenUpgrade Records* to generate and compare
     representations of installed modules on different versions of
     OpenUpgrade server.  Status: stable.
 
-  * A logging facility that dumps the OpenERP database layout in a
+  * A logging facility that dumps the Odoo database layout in a
     format that can be processed further.  It is included in the server
-    distribution.  This functionality has been implemented for OpenERP 5,
+    distribution.  This functionality has been implemented for Odoo 5,
```
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/status.rst:13]: Odoo or OpenERP?

File: openerp/openupgrade/doc/source/xmlids.rst [hunk 3]
```diff
@@ -28,11 +28,11 @@ More interesting are
...
     * res.groups
       If a res groups has moved module (example: hr_manager moved from module
-      hr to base in OpenERP 6), implement this change in the pre-upgrade script
+      hr to base in Odoo 6), implement this change in the pre-upgrade script
       for the module from which it moved.
 
-    * res.roles (migrating from OpenERP 5 to 6 only)
-      These have been migrated to groups in OpenERP6. Find out which group has
+    * res.roles (migrating from Odoo 5 to 6 only)
+      These have been migrated to groups in Odoo 6. Find out which group has
```
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/xmlids.rst:35]: Odoo or OpenERP?

Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/xmlids.rst:35]: I'd say Odoo

Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/xmlids.rst:35]: Well, the product was called OpenERP in that versions, but it's not so important.

File: openerp/openupgrade/doc/source/intro.rst (hunk 1 above)
Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/intro.rst]: Fixed, tnx

**Comment by pedrobaeza** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**